### DISCUSSION TITLE
NextJS ou PHP

### DISCUSSION THREAD

**Discussion by Victor-02** [frontendbr/forum#2440] [Discussão]
Quero começar a fazer freelances, ja trabalhei com React e Angular, mas nunca tive contato com PHP, gostaria de saber se o NextJS seria melhor que o PHP para sites simples como uma landing page ou um site de apresentação de um profissional.

**Comment by ninetails** [comment]
Depende de que portifólio de serviços você quer oferecer.

Para um site landing page ou institucional, que geralmente não tenha dados trafegados, somente apresentação, com front vc pode fazer muito.

Agora se vc quiser oferecer mais serviços como um sistema um tanto complexo, que guarde dados, que possua login etc, saber de backend te dá um leque a mais para prestar serviços.

Se você utilizar NextJS já tendo trabalhado com React, você tem a vantagem de já ter tido o caminho das pedras pois no Next você utiliza JavaScript e React, algo que vc já tem mais familiaridade.

Sobre PHP, depende muito, como falei, de quais serviços você quer oferecer. Pode ser um desafio mas também pode ser um investimento que pode lhe abrir mais portas. Afinal trabalho freelance nem sempre é só criar do zero, pode ser que vc pegue manutenção de sistemas e PHP sendo bem popular por aqui, saber pode te ajudar.

Lembrando que adicionar um backend talvez possa também adicionar complexibilidade em relação a ter que ir atrás e aprender também sobre hospedagem para rodar sua aplicação, visto que não é mais só jogar os arquivos num servidor em produção.

**Reply by Victor-02** [reply]
Entendi, o PHP é bastante promissor mesmo, vou estudar um pouco dos dois, e pesquisar uns freelas e ver o que eles pedem, ai vou ter uma base legal pra decidir. Muito obrigado
